### PR TITLE
Gh/issue19

### PR DESCRIPTION
**Pull request by tom-schier**
Created the updated definitions for 1.20.0. Included some more conditional property validation, especially where the conditional field uses a $ref

### CODE REVIEW

File: 1.20.0/schemas/energy/EnergyPlanContract.json [hunk 1]
```diff
@@ -8,22 +8,37 @@
       ]
     },
     "controlledLoad": {
-      "$ref": "EnergyPlanControlledLoad.json"
+        "$ref": "EnergyPlanControlledLoad.json"
```
Comment (abeheron) [inline 1.20.0/schemas/energy/EnergyPlanContract.json:11]: Space erroneously introduced?

File: 1.20.0/schemas/dcr/ClientRegistration.json [hunk 2]
```diff
@@ -0,0 +1,46 @@
+{
```
Comment (abeheron1) [inline 1.20.0/schemas/dcr/ClientRegistration.json:1]: There used to be a ClientRegistrationRequest in the 1.19.0 which included the schema part for the jwt string to be decoded, which wasn't included in the dcr folder. It this on purpose?

Comment (tom-schier) [reply 1.20.0/schemas/dcr/ClientRegistration.json:1]: I have added this again.

File: 1.20.0/schemas/energy/EnergyPlanContract.json (hunk 1 above)
Comment (tom-schier) [reply 1.20.0/schemas/energy/EnergyPlanContract.json:11]: This seems to be perfectly aligned in my editor. Lin1 11 starts at Col 7.

**Comment by tom-schier** [comment]
I also had to recreate the PM consolidated files.

**Review by abeheron — APPROVED** [review]

**Review by abeheron — APPROVED** [review]

**Review by abeheron1 — APPROVED** [review]

**Pull request merged**